src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, *, data: dict, code: Path, path: Path, **kwargs):
         self._path = Path(path).resolve()
         # hash of flow's entry file, used to skip invoke if entry file is not changed
         self._content_hash = kwargs.pop("content_hash", None)
-        super().__init__(**kwargs)
+        super().__init__()
```

Comment:
Maybe we can directly throw error if there's unhandled kwargs.

Reply:
Good point.